perf/benchmark/security/generate_policies/generate_policies.go

```diff
@@ -0,0 +1,190 @@
+	for scanner.Scan() {
+		rulesYaml.WriteString(" " + scanner.Text() + "\n")
+	}
+	return string(headerYaml) + rulesYaml.String(), nil
+}
+
+func getOrderedKeySlice(ruleToOccurrences map[string]*ruleOption) *[]string {
+	var sortedKeys []string
+	for key := range ruleToOccurrences {
+		sortedKeys = append(sortedKeys, key)
+	}
+	sort.Strings(sortedKeys)
+	return &sortedKeys
+}
+
+func generateAuthorizationPolicy(action string, ruleToOccurrences map[string]*ruleOption, policy *MyPolicy) (string, error) {
+	spec := &authzpb.AuthorizationPolicy{}
+	switch action {
+	case "ALLOW":
+		spec.Action = authzpb.AuthorizationPolicy_ALLOW
+	case "DENY":
+		spec.Action = authzpb.AuthorizationPolicy_DENY
+	}
+
+	var ruleList []*authzpb.Rule
+	sortedKeys := getOrderedKeySlice(ruleToOccurrences)
+	for _, name := range *sortedKeys {
+		ruleOp := ruleToOccurrences[name]
+		rule, err := ruleOp.gen.generate(name, ruleOp.occurrence, action)
+		if err != nil {
+			return "", err
+		}
+		ruleList = append(ruleList, rule)
+	}
+	spec.Rules = ruleList
+
+	yaml, err := ToYAML(policy, spec)
+	if err != nil {
+		return "", err
+	}
+	return yaml, nil
+}
+
+func generateRule(action string, ruleToOccurrences map[string]*ruleOption,
+	policy *MyPolicy) (string, error) {
+
+	switch policy.Kind {
+	case "AuthorizationPolicy":
+		return generateAuthorizationPolicy(action, ruleToOccurrences, policy)
+	case "PeerAuthentication":
+		fmt.Println("PeerAuthentication")
+		// TODO implement
+		// return generatePeerAuthentication(selector, mtl []*mode, portLevel)
+	case "RequestAuthentication":
+		fmt.Println("RequestAuthentication")
+		// TODO implement
+		// return generateRequestAuthentication(selector, ruleToOccurrences)
+	default:
+		fmt.Println("invalid policy")
+	}
+	return "", fmt.Errorf("invalid policy")
+}
+
+func createRules(action string, ruleToOccurrences map[string]*ruleOption, policy *MyPolicy) (string, error) {
+	yaml, err := generateRule(action, ruleToOccurrences, policy)
+	if err != nil {
+		return "", err
+	}
+	return yaml, nil
+}
+
+func createPolicyHeader(namespace string, name string, kind string) *MyPolicy {
+	metadata := MetadataStruct{namespace, name}
+	return &MyPolicy{
+		ApiVersion: "security.istio.io/v1beta1",
+		Kind:       kind,
+		Metadata:   metadata,
+	}
+}
+
+func createRuleOptionMap(ruleToOccurancesPtr map[string]*int) *map[string]*ruleOption {
+	ruleOptionMap := make(map[string]*ruleOption)
+	for rule, occurrence := range ruleToOccurancesPtr {
+		ruleOptionMap[rule] = &ruleOption{}
+		ruleOptionMap[rule].occurrence = *occurrence
+		switch rule {
+		case "when":
+			ruleOptionMap[rule].gen = conditionGenerator{}
+		case "to":
+			ruleOptionMap[rule].gen = operationGenerator{}
+		case "from":
+			ruleOptionMap[rule].gen = sourceGenerator{}
+		default:
+			fmt.Println("invalid rules")
+		}
+	}
+	return &ruleOptionMap
+}
+
+func main() {
+	namespacePtr := flag.String("namespace", "twopods-istio", "Current namespace")
```

Comment:
Add some more details of what does the current namespace mean, is it used in the from in the policy or is it the namespace of the policy in metadata?